application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -112,4 +112,11 @@ private RestAction<Message> sendHelperHeadsUp(ThreadChannel threadChannel) {
         return threadChannel.sendMessage(headsUpWithoutRole)
             .flatMap(message -> message.editMessage(headsUpWithRole));
     }
+
+    private String getEmbedOrRawMessage(Message message) {
```

Comment:
NIT: the title isnt bad, but i propose changing it regardless.
currently, it describes what the method internally does. but instead, it should reflect the purpose of the method. its supposed to extract the **question content**. so perhaps `getQuestionContent(message)`